lambda/utils.js

```diff
@@ -111,6 +111,23 @@ module.exports.waitForFunctionUpdate = async(lambdaARN) => {
     return lambda.waitFor('functionUpdated', params).promise();
 };
 
+module.exports.waitForAliasActive = async(lambdaARN, alias) => {
+    console.log('Waiting for alias to be active');
+    const params = {
+        FunctionName: lambdaARN,
+        Qualifier: alias,
+        $waiter: {
+            // https://aws.amazon.com/blogs/developer/waiters-in-modular-aws-sdk-for-javascript/
+            // "In v2, there is no direct way to provide maximum wait time for a waiter.
+            // You need to configure delay and maxAttempts to indirectly suggest the maximum time you want the waiter to run for."
+            // 24 is ~2 minutes
+            maxAttempts: 24,
```

Comment:
I'd suggest configuring the delay as well here since the default value isn't well known:

```
delay: 5,
```

Is 5 seconds a good delay for this? is there a min/max/average activation time?

Reply:
good point.  Regarding the delay, I think 5s is probably fine for now since I don't have a good idea on the min/max/average activation time.  No sure if it's consistent or always depends on the function code size.  From the few runs I did it _did_ seem to be consistent across power factors.